lib/svix.ts

```diff
@@ -31,3 +32,25 @@ export const listWebhooks = async (appId: string) => {
 export const deleteWebhook = async (appId: string, endpointId: string) => {
   return await svix.endpoint.delete(appId, endpointId);
 };
+
+export const sendInvitationEvent = async (
```

Comment:
I recommend using a single method to send all types of events instead of creating method event types. With the current approach, you'll create new methods whenever the application introduces a new event type. I hope it makes sense.

You can create a method like 

```
const createEvent = (appId, eventType, payload: any) => {...}
```

Reply:
Yes, I refactored the code accordingly.

Reply:
Thanks @Saptarshee94 LGTM. I'll test it locally to see it actually send Webhook events.